components/mpas-ocean/src/shared/mpas_ocn_gm.F

```diff
@@ -910,7 +955,37 @@ subroutine ocn_GM_init(domain, err)!{{{
             sfcTaperFactor = 0.0_RKIND
          end if
 
+         local_config_GM_constant_bclModeSpeed = config_GM_constant_bclModeSpeed
+
+         if (config_GM_minBclModeSpeed_method=='constant') then
+            gm_minBclModeSpeed_compute_on = 0.0_RKIND
+            gm_minBclModeSpeed_constant = config_GM_constant_bclModeSpeed
+         elseif (config_GM_minBclModeSpeed_method=='computed') then
+            gm_minBclModeSpeed_compute_on = 1.0_RKIND
+            gm_minBclModeSpeed_constant = 0.0_RKIND
+         else
+            call mpas_log_write('config_GM_minBclModeSpeed_method must be constant or computed', &
+                                 MPAS_LOG_CRIT)
+         end if
+
          RediGMinitValue = 1.0_RKIND
+         !compute beta
+         if(config_GM_closure == 'Visbeck'.or. &
+            config_GM_closure == 'visbeck'.or. &
+            config_GM_closure == 'EdenGreatbatch'.or. &
+            config_GM_closure == 'edengreatbatch'.or. &
+            config_GM_closure == 'edenGreatbatch'.or. &
+            config_gm_enable_horizontal_resolution_function) then
+            call mpas_pool_get_config(meshPool, 'sphere_radius', sphere_radius)
+            call mpas_pool_get_array(meshPool, 'latEdge', latEdge)
+            !$omp parallel
+            !$omp do schedule(runtime)
+            do iEdge=1,nEdges
+               betaEdge(iEdge) = 2.0_RKIND*omega*cos(latEdge(iEdge)) / sphere_radius
```

Comment:
since `betaEdge` is already defined at this point, I think we can remove it from the two if statements later in the code, the ones for Visbeck and EdenGreatbatch? (that's something that I had noticed before, looking at Luke's maps_ocn_gm changes).

Reply:
Agreed. I'll remove those later two.